.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Release
+
+on: workflow_dispatch # TODO set this on a deploy schedule
```

Comment:
Note: this is so you can "press a button and deploy" later we change this to your deploy schedule

Reply:
Only Uniswap team members can click this button, right? :p